workers/unity/Assets/link.xml

```diff
@@ -1,6 +1,6 @@
 <linker>
 	<assembly fullname="Improbable.Gdk.Core" preserve="all"/>
-    <assembly fullname="Improbable.Gdk.GameObjectRepresentation.GameObjectCreation" preserve="all"/>
+    <assembly fullname="Improbable.Gdk.GameObjectCreation" preserve="all"/>
```

Comment:
Super nit: spacing alignment

Reply:
fixed spacing for the other lines :trollface: